docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Containers are not comunicating between each other

Reply:
Docker containers run by default on the same network, I think:

https://docs.docker.com/compose/networking/:
`By default Compose sets up a single network for your app. Each container for a service joins the default network and is both reachable by other containers on that network, and discoverable by them at a hostname identical to the container name.`

Is this not the situation you are referring to?

Reply:
Yes, but I didn't actually get the json in the UI. Now, with the Echo server, I do, maybe it was a CORS problem. Solved now ;)